content/en/docs/setup/install/helm/snips.sh

```diff
@@ -29,10 +29,73 @@ snip_install_base() {
 helm install istio-base istio/base -n istio-system
 }
 
+snip_installation_steps_3() {
```

Comment:
Shouldn't you add calls to these new snips in test.sh to confirm that they work as expected?

Reply:
Added some of them, but for others major steps were already covered